config/routes.rb

```diff
@@ -22,6 +22,7 @@
     resources :accounts, only: [:index]
     resources :additional_accounts, only: %i[index create new update]
     resource :own_home, only: %i[show update]
+    resources :restrictions, only: %i[index create] # as multiple restrictions
```

Comment:
why `update` instead of `create` wouldn't work?

Reply:
As I am using `resources` here, `update` would require an `:id` within the url. So I've used `create` as that does not require the `:id`.

The alternative would be to use `resource` but then the url ends `\restriction` and I think it should be `\restrictions` (plural). I could use override the path construction to return a plural with `resource` here, but using the `:index, :create` pair here rather than `:show, :update` seemed to make sense to me - especially as multiple restrictions are being managed by these controller actions.